tests/model/eventsources/test_mq_event_source.py

```diff
@@ -40,3 +41,49 @@ def test_get_policy_statements(self):
             }
         ]
         self.assertEqual(policy_statements, expected_policy_document)
+
+    def test_must_validate_secrets_manager_kms_key_id(self):
+        self.mq_event_source.SourceAccessConfigurations = [{"Type": "BASIC_AUTH", "URI": "SECRET_URI"}]
+        self.mq_event_source.Broker = "BROKER_ARN"
+        self.mq_event_source.SecretsManagerKmsKeyId = ["1abc23d4-567f-8ab9-cde0-1fab234c5d67"]
```

Comment:
Could we parameterize this to include anything that is not a string? We covered list here but would be good to add int, bool and dict to add move coverage.